docs/reference/docs/data-replication.asciidoc

```diff
@@ -5,12 +5,12 @@
 [float]
 === Introduction
 
-Each index in Elasticsearch is <<getting-started-shards-and-replicas,divided into shards>>
+Each index in Elasticsearch is <<scalability,divided into shards>>
 and each shard can have multiple copies. These copies are known as a _replication group_ and must be kept in sync when documents
 are added or removed. If we fail to do so, reading from one copy will result in very different results than reading from another.
 The process of keeping the shard copies in sync and serving reads from them is what we call the _data replication model_.
 
-Elasticsearch’s data replication model is based on the _primary-backup model_ and is described very well in the 
+Elasticsearch’s data replication model is based on the _primary-backup model_ and is described very well in the
```

Comment:
```suggestion
{es}'s data replication model is based on the _primary-backup model_ and is described very well in the
```

We may want to consider replacing emphasis (italics) here with bold. I think it's a bit more readable.